src/Compilers/CSharp/Portable/Emitter/Model/PEModuleBuilder.cs

```diff
@@ -1421,12 +1438,29 @@ internal SynthesizedAttributeData SynthesizeIsReadOnlyAttribute(Symbol symbol)
             return TrySynthesizeIsReadOnlyAttribute();
         }
 
+        internal SynthesizedAttributeData SynthesizeIsByRefLikeAttribute(Symbol symbol)
+        {
+            if ((object)Compilation.SourceModule != symbol.ContainingModule)
+            {
+                // For symbols that are not defined in the same compilation (like NoPia), don't synthesize this attribute.
+                return null;
```

Comment:
Yes we have.

---
In reply to: [118795197](https://github.com/dotnet/roslyn/pull/19795#discussion_r118795197) [](ancestors = 118795197)